config/default/kustomization.yaml

```diff
@@ -22,13 +22,13 @@ commonLabels:
   app.kubernetes.io/name: kusk-gateway
 
 patchesStrategicMerge:
-- manager_auth_proxy_patch.yaml
-- manager_webhook_patch.yaml
+  - manager_auth_proxy_patch.yaml
+  - manager_webhook_patch.yaml
 
 images:
-- name: kusk-gateway
-  newName: kubeshop/kusk-gateway
-  newTag: v1.3.20
-- name: kusk-gateway:dev
-  newName: kubeshop/kusk-gateway
-  newTag: latest
+  - name: kusk-gateway
+    newName: kubeshop/kusk-gateway
+    newTag: v1.3.21
```

Comment:
Let's not bump this without building manifest_data. Can you please revert this for now

Reply:
Resolved.

I've reverted the change.